cmd/cluster-kube-scheduler-operator/render/render.go

```diff
@@ -117,10 +116,6 @@ func (r *renderOpts) Validate() error {
 }
 
 func (r *renderOpts) complete() error {
```

Comment:
is this function for future now?

Reply:
yeah, also complete() is something all commands implement....